internal/util.go

```diff
@@ -290,3 +294,77 @@ func setUnitValue(param string, value string) error {
 
 	return nil
 }
+
+func setUnitValueWithinFlatpak(param string, value string) error {
+	hostSpawnCmd := "host-spawn"
+	hostCommandArgs := []string{"sudo", "sh", "-c"}
+	shellCmd := fmt.Sprintf("echo %v | tee %v", value, UnitMatrix[param])
+	hostCommandArgs = append(hostCommandArgs, shellCmd)
+	fmt.Println("Executing command ", hostSpawnCmd, hostCommandArgs)
+	cmd := exec.Command(hostSpawnCmd, hostCommandArgs...)
+	stdin, err := cmd.StdinPipe()
+	if err != nil {
+		fmt.Println(err)
+		return err
+	}
+	cmd.Start()
+	_, err = stdin.Write([]byte(CryoUtils.UserPassword + "\n"))
+	if err != nil {
+		fmt.Println(err)
+		return err
+	}
+	return cmd.Wait()
+}
```

Comment:
All of these functions would need to be documented prior to being ready for prime-time.